lib/components/pop-up/index.tsx

```diff
@@ -15,6 +15,7 @@ interface PopUpProps {
   confirmText?: string;
   onConfirm: () => void;
   style?: React.CSSProperties;
+  size?: 'lg' | 'sm' | 'md' | 'xl' | '2xl';
```

Comment:
Do we need to specify all of these sizes yet? Aren't we just using 2xl so far? It may be a good idea to limit code to only what we need to use

Reply:
I feel like it couldn't hurt to have all these values since stuff may change in the future. Plus, all the values listed in the enum are supported by chakra ui's modal component so nothing gonna break if we ever switch from 2xl to another value.